onnx/defs/schema.h

```diff
@@ -495,10 +495,27 @@ class OpSchemaRegistry {
   // Domain with default value "" means ONNX.
   static const OpSchema* Schema(
       const std::string& key,
-      const std::string& domain = "") {
+      const std::string& domain = "",
+      const int domain_version = -1) {
     auto& m = map();
+    // TODO: Better to use find() to avoid doing the lookup twice
+    // Invariant: any map in the schema is non-empty
     if (m.count(key) && m[key].count(domain)) {
-      return &m[key][domain].rbegin()->second;
+      if (domain_version < 0) {
+        // use latest
+        return &m[key][domain].rbegin()->second;
+      } else {
+        auto& submap = m[key][domain];
+        auto next_it = submap.upper_bound(domain_version);
```

Comment:
I added the function below to return operator schema with newest but not greater than specified version. Is that what you want please?

Reply:
Yes it looks like they are equivalent.